stdlib/messaging/kafka/src/test/resources/test-src/security/sasl_plain_consumer.bal

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/kafka;
+
+kafka:AuthenticationConfiguration authConfigValid = {
+    authenticationMechanism: kafka:AUTH_SASL_PLAIN,
+    username: "ballerina",
+    password: "ballerina-secret"
+};
+
+kafka:AuthenticationConfiguration authConfigInvalidPassword = {
+    authenticationMechanism: kafka:AUTH_SASL_PLAIN,
```

Comment:
Are we support any other mechanism? 
Do we have the same `username`, `password` config for the other authentication mechanisms

Reply:
Yes, SASL/SCRAM is is going to be supported (Implemented locally, will send the PR once this gets merged). It also uses the `username` and the `password`